### PR TITLE
Create nodejs.yml

### PR DESCRIPTION
**Pull request by stannlee**

### CODE REVIEW

**Comment by stannlee** [comment]
@azim254  and @BILLthebuilder  Do you think this workflow is correct?

**Comment by azim254** [comment]
describe the steps to the workflow what it does

**Comment by stannlee** [comment]
> describe the steps to the workflow what it does

Please read the nodejs.yml file

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (BILLthebuilder) [inline .github/workflows/nodejs.yml]: You could add version 10.x for compatibility purposes

Comment (stannlee) [reply .github/workflows/nodejs.yml]:
```suggestion
        node-version: [12.x]
```
Okay though it is compatible because we re using express js

**Pull request merged**